src/transforms/FilterToSchema.ts

```diff
@@ -114,6 +114,7 @@ function filterDocumentToSchema(
     });
   });
 
+  const fragmentSet = {};
```

Comment:
Let's make this `= Object.create(null)` so we don't inherit any truthy values from `Object.prototype` by accident.